it/src/main/java/org/corfudb/universe/node/server/AbstractCorfuServer.java

```diff
@@ -72,13 +72,11 @@ public int compareTo(CorfuServer other) {
      * @return maven/project version
      */
     protected static String getAppVersion() {
-        MavenXpp3Reader reader = new MavenXpp3Reader();
-        Model model;
-        try {
-            model = reader.read(new FileReader("pom.xml"));
-            return model.getParent().getVersion();
-        } catch (IOException | XmlPullParserException e) {
-            throw new NodeException("Can't parse application version", e);
+        String version = System.getProperty("project.version");
+        if (version == null || version.isEmpty()){
+            throw new NodeException("Can't get application version. Please specify 'project.version' parameter");
```

Comment:
Probably change the exception description after you fix the "project.version" issue.